src/main/java/stream/flarebot/flarebot/util/general/GuildUtils.java

```diff
@@ -0,0 +1,302 @@
+                    tmp = Getters.getUserById(l);
+                } else {
+                    Member temMember = Getters.getGuildById(guildId).getMemberById(l);
+                    if (temMember != null) {
+                        tmp = temMember.getUser();
+                    }
+                }
+                if (tmp != null) {
+                    return tmp;
+                } else if (forceGet) {
+                    return Getters.retrieveUserById(l);
+                }
+            } catch (NumberFormatException | NullPointerException ignored) {
+            }
+        }
+        return null;
+    }
+    //Channel
+
+    /**
+     * Gets a {@link TextChannel} from a string
+     *
+     * @param arg The string to get the channel from
+     * @return null if the channel couldn't be found otherwise a {@link TextChannel}
+     */
+    public static TextChannel getChannel(String arg) {
+        return getChannel(arg, null);
+    }
+
+    /**
+     * Gets a {@link TextChannel} from a string
+     *
+     * @param channelArg The string to get the channel from
+     * @param wrapper The Guild wrapper for the {@link Guild} that you want to get the channel from
+     * @return null if the channel couldn't be found otherwise a {@link TextChannel}
+     */
+    public static TextChannel getChannel(String channelArg, GuildWrapper wrapper) {
+        try {
+            long channelId = Long.parseLong(channelArg.replaceAll("[^0-9]", ""));
+            return wrapper != null ? wrapper.getGuild().getTextChannelById(channelId) : Getters.getChannelById(channelId);
+        } catch (NumberFormatException e) {
+            if (wrapper != null) {
+                List<TextChannel> tcs = wrapper.getGuild().getTextChannelsByName(channelArg, true);
+                if (!tcs.isEmpty()) {
+                    return tcs.get(0);
+                }
+            }
+            return null;
+        }
+    }
+    //Emote
+
+    /**
+     * Gets an {@link Emote}
+     *
+     * @param l the id of the emote
+     * @return null if the id is invalid otherwise a {@link Emote}
+     */
+    public static Emote getEmoteById(long l) {
+        return Getters.getGuilds().stream().map(g -> g.getEmoteById(l))
+                .filter(Objects::nonNull).findFirst().orElse(null);
+    }
+
+    /**
+     * Gets weather or not the bot can change nick
+     *
+     * @param guildId The guild to check if we can change nick
+     * @return if we change change nick
+     */
+    public static boolean canChangeNick(String guildId) {
+        return Getters.getGuildById(guildId) != null &&
+                (Getters.getGuildById(guildId).getSelfMember().hasPermission(Permission.NICKNAME_CHANGE) ||
+                        Getters.getGuildById(guildId).getSelfMember().hasPermission(Permission.NICKNAME_MANAGE));
+    }
+
+    /**
+     * Joins the bot to a channel.
+     *
+     * @param channel The chanel to join the bot to.
```

Comment:
This is actually false, it is a text channel and is used for error messages. The channel we is from the member.